initrd/options.go

```diff
@@ -41,3 +46,11 @@ func WithArchitecture(arch string) InitrdOption {
 		return nil
 	}
 }
+
+// WithKraftignorePath sets the path for .kraftignore.
+func WithKraftignorePath(dir string) InitrdOption {
+	return func(opts *InitrdOptions) error {
+		opts.kraftignorePath = filepath.Join(dir, KraftignoreFileName)
+		return nil
+	}
+}
```

Comment:
As a result of it being implicitly available, we can remove this option method.

Reply:
Done 👍🏽